includes/class-aftership-actions.php

```diff
@@ -519,6 +546,52 @@ public function delete_tracking_item( $order_id, $tracking_id ) {
 		return $is_deleted;
 	}
 
+    public function delete_fulfillment( $order_id, $fulfillment_id )
+    {
+        $fulfillments = $this->get_fulfillments_by_wc($order_id);
+		$deleted_fulfillment = null;
+
+        if ( count( $fulfillments ) > 0 ) {
```

Comment:
可以尝试使用 array_filter 这个函数。

```suggestion
$filteredArray = array_filter($array, function($item, $index) {
    return $item['id'] == $fulfillment_id;
}, ARRAY_FILTER_USE_BOTH);
```